Until recent developments in the last decade, DNA sequencing has been carried out under laborious and low-throughput techniques. The so-called next-generation sequencing (NGS) distinguishes itself from older techniques by massively scaling up the process of DNA sequencing in a time- and cost-effective manner. NGS has transformed the field of genomics and by extension, transcriptomics, as it enables biological questions to be addressed on a molecular and genome-wide level. This thesis focuses on the analysis of transcriptomes through the use of NGS and bioinformatics. While the genome is more or less identical in the cells that make up an organism, their transcriptomes are markedly different. Cells in different physiological states express a different cohort of transcripts. The study of transcriptomes allows us to understand the molecular mechanisms that drive biological processes and those that define cellular identity or disease status.

This thesis is composed of six genome-wide transcriptome studies. The first study was a technical analysis on biases introduced through the use of molecular barcoding of samples prior to sequencing. The second study investigated whether small RNAs were involved in the DNA damage response through small RNA sequencing. The third study analysed the global effects of \textit{Mecp2} knockout on a class of small RNAs called Piwi-interacting RNA. The fourth study examined transcriptome differences between two different cell culturing conditions on human induced pluripotent stem cells. The fifth study was an analysis of the expression patterns of repetitive elements in the FANTOM5 samples. The sixth study examined the whole blood transcriptomes of Parkinson's disease patients in contrast to healthy controls. Collectively, these studies demonstrated the applicability of transcriptome sequencing and bioinformatics in gaining insight into various biological problems.

NGS has transformed the field of transcriptomics and revealed a much more complex picture of transcript expression in various biological systems. Unravelling the complexity of the transcriptome will only be possible with an in-depth understanding of the technology, of the intrinsic properties of the dataset, and through the use of appropriate bioinformatic methods.
