Until recent developments, DNA sequencing has been carried out under laborious and low-throughput techniques. The so-called next-generation sequencing (NGS) distinguishes itself from older techniques by scaling up the process of DNA sequencing but with a reduction in the cost and time. NGS has transformed the field of genomics and by extension, transcriptomics, as it enables biological questions to addressed on a molecular and genome-wide level. This thesis is on the analysis of transcriptomes through the use of NGS and bioinformatics. While the genome is more or less identical in the cells that make up an organism, their transcriptomes are markedly different. Cells under different physiological states express a different cohort of transcripts. The study of transcriptomes allows us to understand the molecular mechanisms that are behind biological processes and those that drive cellular identity or disease.

This thesis is composed of 6 genome-wide transcriptome studies. The first study was a technical analysis on a type of bias introduced by the molecular barcoding of samples prior to sequencing. The second study investigated whether small RNAs were involved in the DNA damage response through small RNA sequencing. The third study analysed the global effects of \textit{Mecp2} knockout on a class of small RNAs called Piwi-interacting RNA. The fourth study examined transcriptome differences between two different cell culturing conditions on human induced pluripotent stem cells. The fifth study was an analysis of the expression patterns of repetitive elements in the FANTOM5 samples. The sixth study examined the whole blood transcriptomes of Parkinson's disease patients in contrast to healthy controls. Collectively, these studies demonstrated the applicability of transcriptome sequencing and bioinformatics in gaining insight into various biological problems.

The applications of NGS are essentially limited only by the imaginations of researchers and the ability to make sense of the large quantity of sequencing data. As these two factors develop, the complexity of transcriptome may be further unravelled.
