Until recent developments in the last decade, DNA sequencing has been carried out under laborious and low-throughput techniques. The so-called next-generation sequencing (NGS) distinguishes itself from older techniques by massively scaling up the process of DNA sequencing in a time- and cost-effective manner. NGS has transformed the field of genomics and by extension, transcriptomics, as it enables biological questions to be addressed on a molecular and genome-wide level. This thesis focuses on the analysis of transcriptomes through the use of NGS and bioinformatics. While the genome is more or less identical in the cells that make up an organism, their transcriptomes are markedly different. Transcriptomics allows us to characterise the functional elements of the genome, revealing the cohort of transcripts that define cells and tissues, which will help better understand molecular mechanisms that drive development as well as disease.

Various technologies have been developed to utilise NGS for the cataloguing and quantifying of transcripts from cells and tissues. These technologies are relatively immature, as they have been developed within the last 5-6 years, and as such are constantly being investigated and improved upon. In the first study of this thesis, biases and technical artefacts in a technology known as nano cap analysis gene expression (nanoCAGE) were investigated. We found that biases were introduced through the use of molecular barcodes and developed several strategies for coping with such biases. One of the advantages of using NGS for transcriptome profiling is the ability to detect transcripts without any \textit{a priori} knowledge of genomic sequences. In the second study, the transcriptional output of cells with induced DNA damage was captured using small RNA sequencing. We observed a previously uncharacterised class of RNAs that formed near a DNA break site and were necessary in establishing the DNA damage response. This study was the first to suggest that small RNAs are necessary for DNA damage repair. In the third and last study, we focused on transcripts that initiated from repetitive elements (REs) and characterised their expression patterns across a wide panel of cell lines, tissues, and primary cells. We demonstrated that REs may drive the expression of long non-coding RNAs and enhancer RNAs, and their expression patterns are tissue specific. These findings suggest that not all transcription events arising from REs are simply random transcription events or transcriptional noise.

NGS has transformed the field of transcriptomics by revealing a much more complex picture of transcription than previously anticipated. 
