Until recent developments in the last decade, DNA sequencing has been carried out under laborious and low-throughput techniques. The so-called next-generation sequencing (NGS) distinguishes itself from older techniques by massively scaling up the process of DNA sequencing in a time- and cost-effective manner. NGS has transformed the field of genomics and by extension, transcriptomics, as it enables biological questions to be addressed on a molecular and genome-wide level. This thesis focuses on the analysis of transcriptomes through the use of NGS and bioinformatics. While the genome is more or less identical in the cells that make up an organism, their transcriptomes are markedly different. Cells in different physiological states express a different cohort of transcripts. The study of transcriptomes allows us to understand the molecular mechanisms that drive biological processes and those that define cellular identity or disease status.

Transcriptional output as measured by NGS, is influenced mainly by technical and biological factors, and it is important to address technical factors prior to making biological conclusions. Biases and technical artefacts may arise due to the experimental protocol and create spurious results. In the first study, biases introduced through the use of molecular barcodes were identified and alleviated by a computational method and by an improved experimental protocol. In the second study, the transcriptional output of small RNAs, in response to DNA damage, were characterised. We observed a specific class of RNAs that formed near a DNA break site and were necessary in establishing the DNA damage response. The third study analysed transcripts that would initiate within repetitive elements and their expression patterns were characterised across a wide panel of cell lines, tissues, and primary cells. Transcription from these loci have been largely dismissed as simply random transcriptional events, i.e transcriptional noise, however there were examples where there was evidence for regulation.

NGS has transformed the field of transcriptomics by revealing a much more complex picture of transcription in various biological systems. Within this complexity lies technical biases, transcriptional noise, and signal arising from biological phenomena. Unravelling this complexity, requires a deep understanding of the technologies used to generate the data and knowledge of the appropriate bioinformatic method.
