Recent developments in DNA sequencing, has led to the development of high-throughput sequencing, which has massively scaled up the process of DNA sequencing in a time- and cost-effective manner. With the completion of the various mammalian genome projects, the focus has shifted towards identifying functional elements in genomes. The field of transcriptomics aims to identify, annotate, and analyse the transcribed products of the genome. The application of high-throughput sequencing to transcriptome profiling has allowed accurate unbiased profiling of transcripts and revealed the complexity of mammalian transcriptomes. However, these applications are relatively immature, as they have been developed within the last 5-6 years, and therefore are constantly being investigated and improved upon. This thesis focuses on the analysis of transcriptomes through the use of high-throughput sequencing and bioinformatic methods.

In the first study of this thesis, biases and technical artefacts in a transcriptome technology known as nano cap analysis gene expression (nanoCAGE) were investigated. We found that biases were introduced through the use of molecular barcodes and developed several strategies for coping with such biases. In the second study, we captured and analysed the transcriptional output of cells with induced DNA damage using small RNA sequencing. We observed a previously uncharacterised class of RNAs that formed near the DNA break site and were necessary in establishing the DNA damage response. In the third and last study, we focused on transcripts that initiated from repetitive elements (REs) and characterised their expression patterns across a wide panel of cell lines, tissues, and primary cells. We demonstrated that REs may drive the expression of long non-coding RNAs and enhancer RNAs, and their expression patterns are tissue specific.

High-throughput sequencing has transformed the field of transcriptomics by revealing a much more complex picture of transcription than previously anticipated. The layers of complexity may include technical artefacts, random transcriptional events or transcriptional noise, or real biological signal. In order to separate noise from signal requires an understanding of the different technologies, careful scrutiny of the data, and the application of appropriate bioinformatic methods.
