The mammalian genome encodes both coding and non-coding transcripts that work in concert to orchestrate a wide range of cellular functions. By profiling the complete set of transcripts expressed, i.e., the transcriptome, the dynamics between transcripts can be inferred.

This thesis focuses primarily on the study of transcriptomes using high-throughput sequencing and bioinformatics. While the genome is assumed to be more or less identical in the 37 trillion cells that make up the human body\cite{pmid23829164}, their transcriptomes are markedly different. It is the differential use of the genome that makes it possible to give rise to over 400 different cell types\cite{pmid16790079} that make up different tissues, organs, and systems. Differential usage may also be indicative of disease; for example, transcriptomes isolated from cancerous cells are quite different from normal cells. In this work, we have studied and compared transcriptomes from various biological samples and systems: in whole blood samples (Chapter ~\ref{template_switching} and ~\ref{blood}), in the FANTOM5 panel of samples (Chapter ~\ref{repeat}), in a DNA damage response (DDR) system (Chapter ~\ref{ddrna}), in a mouse model for Rett syndrome (Chapter ~\ref{pirna}), and in human induced pluripotent stem cells (iPSCs) (Chapter ~\ref{ccl2}). Collectively, these studies demonstrate the applicability of transcriptome sequencing and bioinformatics in gaining insight in various biological systems.
