The work carried out as part of this thesis had the underlying goal of interpreting the transcriptional output from different biological systems. In the various studies, different transcriptome profiling methods employing the use of high-throughput sequencers were used to catalogue and quantify the expression of transcripts. While the genome is largely static, the transcriptome is highly dynamic and the cohort of expressed transcripts at a specific development stage or physiological condition will allow us to understand development and disease on a molecular level. However, two problems complicate the study of transcriptomes: 1) the presence of artefacts and biases and 2) pervasive transcription. In order to properly interpret transcriptional output, it is necessary to filter out the noise from the biological signal.

The identification of artefacts and biases is important prior to the analysis of data; while tools\cite{pmid19737799, fastqc} are available for performing quality control steps, they are unable to deal with idiosyncratic biases from specific protocols. In chapter ~\ref{template_switching}, we demonstrated a particular type of bias that was introduced with the use of molecular barcodes. DNA barcodes are used to allow different RNA sources to be pooled at an early stage of sample preparation and are easily introduced by modifying the sequences of primers or linkers. This has two main benefits in that the starting material is increased and different samples can be sequenced on a single flowcell, increasing the cost-effectiveness. In our study, we observed that the efficiency of primer annealing was dependent on the barcode sequence that was chosen\cite{Tang01022013}, creating a barcode bias. Furthermore, we noticed a particular type of artefact that was artificially produced at the template-switching step in the experimental protocol\cite{Tang01022013}. Specifically, these template-switching artefact, which are a result of strand invasion, interrupted the process of reverse transcription when the cDNA has increased sequence complementary to the template-switching primer. As part of this work, we developed a computational method to remove these artefacts and improved the experimental protocol.

There have been several surprises in the genomics field since the completion of the draft human genome sequence\cite{lander2001initial,venter2001sequence}. The first was the number of genes in the human genome, which many genome biologists had over-estimated\cite{genesweep}. The next surprise was the observation that most of the human genome was transcribed\cite{pmid11988577, pmid17571346}, which became known as the phenomenon of pervasive transcription. While there have been claims that the genome is not as pervasively transcribed as suggested by tiling arrays\cite{pmid20502517}, various transcriptome profiling methods have demonstrated that the human genome is indeed pervasively transcribed\cite{pmid22955620}. The current question is whether these transcriptional products are functional or not. One explanation for pervasive transcription is the low fidelity of RNA polymerase II\cite{pmid17277804}, which may be known as leaky transcription. As these events can potentially interfere with cellular events, they are suppressed by various systems, such as RNA degradation pathways. However, cells with defective RNA degradation machinery show an increase in the number of pervasive transcribed transcripts\cite{pmid24267449}. The question therefore is whether an element that was bound for degradation, could be a functional product; ultimately, only appropriately designed experiments can answer the question.

In chapter ~\ref{ddrna}, we profiled the small RNA population of cells that were induced for DNA damage. A role for small RNAs in the DNA damage response (DDR) had been previously observed in \textit{Neurospora crassa}, where small RNAs were generated from ribosomal RNA repeats when induced with DNA damage\cite{pmid19444217}. The DDR is a coordinated set of events that repairs lesions in DNA; the DNA break is detected by specialised factors that initiates a signalling cascade from the site of DNA damage\cite{pmid19847258}. Our work and the work of an independent group, showed that small RNAs would form in the vicinity of the double strand break\cite{francia2012site,pmid22445173}. We demonstrated that Dicer- and Drosha-dependent small RNAs are required for DDR activation and had size and nucleotide profiles that differed from other classes of small RNAs. Mutations in \textit{Ago2} demonstrated that DNA repair efficiency was impaired\cite{pmid22445173}, and the authors suggested that AGO2 recruits the DNA damage repair complex. While argonaute proteins are known to bind different classes of small RNAs, they are mainly responsible for RNA interference. It is therefore unclear what role AGO2 and these small RNAs play in the DDR.

Pervasive transcription also implies that repetitive elements are transcribed, since the human genome is largely occupied by repetitive DNA sequences, made up of mainly transposable elements (TEs). A large number of transcriptional events was indeed shown to be initiated within TEs\cite{pmid19377475}. In chapter ~\ref{repeat}, we further profiled the expression of TEs in a large panel of samples to capture tissue-specific expression patterns. As previously reported, we observed that on average, expression from TEs are more tissue-specific with lower expression levels than protein-coding transcripts. This can be attributed to the nuclear localisation of transcripts arising from TEs\cite{pmid24777452}. Overlaying the transcriptional events arising from TEs with other genomic features, revealed that they overlap enhancer regions and long non-coding RNAs (lncRNAs). Histone patterns associated with regulatory regions revealed that hypomethylated TEs may serve as enhancer regions\cite{pmid23708189}; furthermore, activated TEs, specific to stem cells, also showed specific enhancer marks\cite{pmid24777452}. The association of TEs with lncRNAs has been previously observed, where a large fraction of the lncRNA sequence is composed of TEs\cite{pmid25218058, pmid23181609}.

Various criteria have been used as a criteria for functionality, in contrast to noise, in transcriptional products; these can be broadly separated into \textit{in silico} and experimental validations. \textit{In silico} methods are based on analysing properties associated with an expressed transcript. The expression strength is a commonly used criteria, as consistently captured signal is less likely to occur just by chance, i.e. the smaller the margin of error with an increase in signal. Context-specific transcription, such as observed expression at a specific time-point, developmental stage, or tissue type, can also be used to support function. The argument is such that a random transcript would not become active under a specific condition. However, the criteria of context-specific expression patterns can be countered by the fact that different cell types and during different developmental stages, express a specific repertoire of transcription factors and are under certain circumstances that would lead to the context-specific transcription. Sequence conservation is usually used to support functionality, as this is evidence for purifying or negative selection. For example, protein-coding genes are typically very well conserved among different organisms. Lastly, transcriptional data is typically shown in conjunction with other genome-wide assays.

The classical view of transcription initiation was that transcription began at a single position at TA-rich regions known as the TATA-box. One of the major findings made using the CAGE technology was that not all transcription initiation events occurred at a single position\cite{pmid16645617}. While the classical TATA-box promoters mostly initiated from a single position (which were termed sharp promoters), promoters that were CG-rich (in particular CpG islands) showed initiation events across a stretch of sequence (these were termed broad promoters). Initially these CAGE tags were thought to be noise, however, these TSSs were highly consistent in orthologous mouse and human promoters, and sharp and broad promoters were consistently detected in various libraries\cite{pmid16645617}. In addition, this initial survey of the transcriptional landscape in mammalian genomes identified many novel mRNAs and non-coding RNAs that had not been previously characterised\cite{pmid16141072}. CAGE has also been applied to study the dynamics of TSS usage throughout a time course of growth arrest and differentiation\cite{pmid19377474}.
