The primary aim of transcriptomics is to capture in an unbiased manner the full set of transcripts in a cell or tissue and their expression patterns\cite{pmid19015660}. Transcriptional activity is different amongst cells and tissues due to the wide range of physiological, biochemical, and developmental differences. In the work presented in this thesis, several different experimental protocols were used to capture the transcriptomes of different biological samples: nanoCAGE\cite{pmid20543846} (Chapter ~\ref{template_switching} and ~\ref{blood}), heliscopeCAGE\cite{pmid21596820} (Chapter ~\ref{repeat} and ~\ref{ccl2}), small RNA sequencing\cite{pmid20964636} (Chapter ~\ref{ddrna} and ~\ref{pirna}), and illuminaCAGE\cite{Takahashi2012} (Chapter ~\ref{blood}). The purpose of each study was to characterise and quantify the differences in transcript expression that drove and specified variation.

In an experimental setting, a single condition or phenomena is under study; for example, the investigation of the effects caused by a protein, such as CCL2 (Chapter ~\ref{ccl}). However, there are many conditions that cannot be controlled or are even unknown. For this particular reason, technical and biological replicates are used. Technical replicates are used to assess the amount of variability that is caused from technical aspects of the experiment. For example, in an early benchmark of RNA-Seq using the Solexa/Illumina sequencing technology, replicate sequencing runs on the same samples demonstrated that technical variations from sequencing were small\cite{pmid18550803}. The work in chapter ~\ref{template_switching}, demonstrated how varying technical aspects of an experiment (in this case the molecular barcode sequence), revealed an experimental biased caused by strand invasion\cite{Tang01022013}. Biological replicates are used to estimate the biological variation between samples that are either under or have the same condition. Biological variation arises from the natural variability inherit amongst different biological systems; this variation may be small for genetically identical samples or very large in heterogeneous samples such as blood (Chapter ~\ref{blood}). Biological processes are also known to be noisy, such as splicing, which increases the mRNA isoform diversity in human cells\cite{pmid21151575}. By using biological replicates as a reference, differences between the condition under study can be assessed. Differential expression is only significant if the conditional difference is sufficiently larger than the biological variation.

The first indication of the importance of non-polyadenylated transcripts\cite{pmid4213457}.

Major portions of eukaryotic genomes are occupied by DNA sequences whose transcripts do not code for proteins. The historical use of poly-A tail selection for enriching mRNAs and removing rRNA contamination in RNA preparations has led to the under-sampling of non-polyadenylated transcripts. The use of random primers in the CAGE protocols, circumvents this bias and allows the capture of poly-A+ and poly-A- RNAs. To investigate the non-coding portion of the genome the CAGE technology was cruicial.

In the case of small RNA sequencing, RNAs are isolated by size and are directly sequenced after adaptor ligation.

Starting RNA amount required by different protocols and it may not be possible obtain sufficient amounts of starting material. Thus PCR-based techniques are popular and are able to amplify RNA isolated from single cells. This leads to less heterogeneity compared to techniques that pool cells, which may have very different expression profiles. It is important to obtain a true profile of transcription in cells. PCR amplification is necessary for fluorescent-based imaging. PCR bias from GC and AT frequencies.

\begin{itemize}
   \item New classes of RNA
   \item Bird’s-eye view of the transcriptome to understand global trends
   \item Increase in piRNA expression due to lost of MeCP2
   \item Up-regulation of hypoxia related genes leading to increase pluripotency
   \item Studying the expression patterns of repetitive elements
   \item Development of molecular signatures that can be used as biomarkers
\end{itemize}

Debate between functional elements and junk DNA

Cytoplasmic versus nuclear enrichment. The complexity of nuclear transcripts determined by denaturation-renaturation experiments\cite{pmid5969070}

CAGE versus RNA-Seq versus microarrays\cite{pmid24676093}

Many ncRNA have low evolutionary conservation and are lowly transcribed, and have been dismissed as technical artifacts and/or background transcription that have no biological importance. Alternatively spliced lncRNA\cite{Johnsson2013}.

Are the majority of detected low-level transcription due to technical artifacts and/or background biological noise? Discussion about sequencing depth and saturation. Targeted RNA sequencing reveals the deep complexity of the human transcriptome\cite{pmid22081020}.

Functional transcriptomics in the post-ENCODE era, specifically what is the criteria for functionality

Validation of ncRNA by the use of chemically engineered antisense oligonucleotides, siRNA, shRNA-mediated approaches.

Viable mouse with deletion of ultra-conserved regions\cite{pmid17803355} or Neat1\cite{pmid21444682}.

\begin{itemize}
   \item Complex picture behind transcript expression, the products of the genome
   \item The number of transcripts can be quantified to get some idea of the level of transcript expression
   \item Transcript information may reveal what genes permit stem cells to self renew and differentiate into different cells
   \item Apart from understanding which transcripts are expressed, expression profiling allows the molecular classification of diseases such as cancer, which can lead to the development of biomarker tests
   \item Building gene networks from transcriptional profiles
   \item Sequencing depth and sampling of RNA molecules; absolute transcript quantification will help (such as using unique molecule identifiers and non-PCR based methods)
\end{itemize}

Transcripts of Unknown Function (TUFs). The role of non-coding RNAs. The role of repetitive elements in the genome.

Small non-coding RNAs have been implicated in many biological processes, such as messenger RNA regulation or transposon silencing. We identified a role of small non-coding RNAs in the DNA damage repair mechanism. The inactivation of dicer and drosha, which are required for the biogenesis of small RNAs, leads to a loss of DNA damage repair. We sequenced the small RNAs from cells that had induced DNA damage and found small RNAs arising from the vicinity of the DNA double-strand break. Importantly, synthetic small RNAs mimicking these small RNAs could drive the DNA damage response \cite{francia2012site}.

Epigenetic roles such as guiding chromatin-modifying enzymes to their sites of action or acting as scaffolds.

Rise of regulatory RNA\cite{Morris2014}.

\begin{itemize}
   \item Unbiased profiling of total transcripts
   \item Comparison with different environmental conditions
   \item Gene ontology enrichment
   \item Inaccuracies of gene models
   \item template-free activity of reverse transcriptase, leads to an additional G nucleotide to the 5' end
\end{itemize}

Parkinson’s disease (PD) is a slowly progressive disease in which dopamine neurons in the substantia nigra degenerate undetected for years before clinical symptoms develop. The lack of clinical symptoms highlights the necessity of a laboratory test, such as an assay for biomarkers, which can correlate subjects with PD risk. We have profiled the RNAs in the whole blood sample of PD patients and age-matched controls using high-throughput deepCAGE sequencing. By comparing the RNA profiles between PD patients and controls, we aim to discover novel biomarkers that are present in whole blood, which may be further developed into a non-invasive clinical test for PD.
