The work carried out as part of this thesis had the underlying goal of interpreting the transcriptional output from different biological systems. In the various studies, different transcriptome profiling methods employing the use of high-throughput sequencers were used to catalogue and quantify the expression of transcripts. While the genome is largely static, the transcriptome is highly dynamic and the cohort of expressed transcripts at a specific development stage or physiological condition will allow us to understand development and disease on a molecular level. However, two problems complicate the study of transcriptomes: 1) the presence of artefacts and biases and 2) pervasive transcription. In order to properly interpret transcriptional output, it is necessary to filter out the noise from the biological signal.

Transcriptome profiling falls under the category of functional genomics, in which the aim is to identify functional elements in the genome by evidence of transcription. The "read out" from such experiments represents the expression of transcripts, which by itself is not entirely informative. The raw signal needs to be annotated or in order words, put into perspective, usually with respect to the genome. In the case of transcriptome profiling via high-throughput sequencing, the raw signal, i.e. sequenced reads, are first processed and mapped back to the genome. The processing step involves quality control steps that remove reads with missing features, ambiguously called bases, and reads that are composed of the primer or adaptor sequences used in the library preparation. Furthermore, different thresholds are usually implemented after read mapping to remove potentially spurious events; these threshold can be based on the mapping confidence or mapping quality, and expression strength. The mapped reads are then annotated to genomic features, such as protein-coding genes and repetitive elements, or to genomic properties, such as sequence conservation to other genomes and the nucleotide composition of a region. Finally, the annotated data is interpreted with respect to the experiment design; ``sanity checks", which are basic tests that can quickly assess the validity of the results, are commonly performed.

The identification of artefacts and biases is important prior to the analysis of data; while tools\cite{pmid19737799, fastqc} are available for performing quality control steps, they are unable to deal with idiosyncratic biases from specific protocols. In chapter ~\ref{template_switching}, we demonstrated a particular type of bias that was introduced with the use of molecular barcodes. DNA barcodes are used to allow different RNA sources to be pooled at an early stage of sample preparation and are easily introduced by modifying the sequences of primers or linkers. This has two main benefits in that the starting material is increased and different samples can be sequenced on a single flowcell, increasing the cost-effectiveness of the experiment. In our study, we observed that the efficiency of primer annealing was dependent on the barcode sequence\cite{Tang01022013}, thus creating a barcode bias. Furthermore, we noticed a particular type of artefact that was artificially produced during the template-switching step of the experimental protocol\cite{Tang01022013}. Specifically, these template-switching artefact, which are a result of strand invasion, interrupted the process of reverse transcription when the cDNA has increased sequence complementary to the template-switching primer. The bias was identified when libraries prepared with the same or similar barcodes clustered together, regardless of the biological origin of the library. We mitigated this bias by introducing a spacer sequencer into the template-switching primer, which standardised the template-switching efficiency.

Another complication with interpreting transcriptional output is pervasive transcription. There have been two surprises in the genomics field since the completion of the draft human genome sequence\cite{lander2001initial,venter2001sequence}. The first was the number of genes in the human genome, which many genome biologists had over-estimated prior to the sequencing of the human genome\cite{genesweep}. This came as a surprise, as it was thought that organismal complexity was a consequence of the number of genes; however, humans have roughly the same number of genes as \textit{Caenorhabditis elegans}. The next surprise was the observation that most of the human genome was transcribed\cite{pmid11988577, pmid17571346}, i.e. pervasive transcription. This phenomenon was first observed using tiling arrays\cite{pmid17510325} and later by various transcriptome profiling methods\cite{pmid22955620}. However, whether pervasive transcription consists mainly of background transcriptional noise or is of functional importance has been a matter of debate. The low fidelity of RNA polymerase II\cite{pmid17277804} and promiscuous binding of transcription factors\cite{pmid22868264} may give rise to ``leaky" transcription. Furthermore, the existence of various systems that suppress pervasive transcription\cite{pmid24267449} also raises the question of whether these products of pervasive transcription are functional.

Despite questions over their functionality, pervasive transcription is known to produce various classes of non-coding RNAs (ncRNAs)with characteristic features. These include various classes of small RNAs that are associated to promoters\cite{pmid19920851}, long non-coding RNAs (lncRNAs)\cite{pmid24290031}, and long intergenic non-coding RNAs (lincRNA)\cite{pmid23818866}. Evidence of context-specific transcription, e.g. transcription at a specific developmental stage, tissue type, or against a particular stimulus, has been observed in these classes of ncRNA. In chapter ~\ref{ddrna}, we profiled the small RNA population of cells that were induced for DNA damage, as a role for small RNAs in the DNA damage response (DDR) had been previously observed in \textit{Neurospora crassa}\cite{pmid19444217}. Our work and the work of an independent group, showed that small RNAs would form in the vicinity of the double strand break\cite{francia2012site,pmid22445173}. Furthermore, these RNAs, which we named as DDRNAs, had specific size and nucleotide profiles that differed from other classes of small RNAs. While it is unclear what role these DDRNAs serve in the DDR, knock-down of Dicer and Drosha\cite{francia2012site} and mutations in \textit{Ago2}\cite{pmid22445173} impair the DNA repair efficiency. As a signalling cascade is initiated from the site of DNA damage\cite{pmid19847258}, it was hypothesised that AGO2, which are known to bind to different classes of small RNAs, helps recruit the DNA damage repair complex\cite{pmid22445173}.

An emerging theme in the potential role of many ncRNAs is that they function as guides and help direct various complexes to specific genomic loci. Apart from the classical example of miRNAs being associated with the RNA-induced silencing complex, many lincRNAs were shown to be bound to chromatin-modifying complexes and may guide these complexes to regions that need to undergo epigenetic regulation\cite{pmid19571010}. Similarly, various ncRNAs, including the well characterised lncRNAs Xist and HOTAIR, are associated with the Polycomb repressive complex 2 and directs this complex towards silencing genomic loci\cite{pmid23431328}. Intriguingly, many enzymatic members of chromatin remodelling complexes do not contain DNA binding domains but possess RNA binding domains. (Write some more about chromatin and ncRNAs.)

(Missing link here.)

Pervasive transcription also implies that repetitive elements are transcribed, since the human genome is largely occupied by repetitive DNA sequences, made up of mainly transposable elements (TEs). A large number of transcriptional events was indeed shown to be initiated within TEs\cite{pmid19377475}. In chapter ~\ref{repeat}, we further profiled the expression of TEs in a large panel of samples to capture tissue-specific expression patterns. As previously reported, we observed that on average, expression from TEs are more tissue-specific with lower expression levels than protein-coding transcripts. This can be attributed to the nuclear localisation of transcripts arising from TEs\cite{pmid24777452}. Overlaying the transcriptional events arising from TEs with other genomic features, revealed that they overlap enhancer regions and lncRNAs. Histone patterns associated with regulatory regions revealed that hypomethylated TEs may serve as enhancer regions\cite{pmid23708189}; furthermore, activated TEs, specific to stem cells, also showed specific enhancer marks\cite{pmid24777452}. The association of TEs with lncRNAs has been previously observed, where a large fraction of the lncRNA sequence is composed of TEs\cite{pmid25218058, pmid23181609}.

(Close on the difficulty of working with the non-coding regions of the genome.)
