The work carried out as part of this thesis had the underlying goal of interpreting the transcriptional output from different biological systems. In the various studies, different transcriptome profiling methods employing the use of high-throughput sequencers were used to catalogue and quantify the expression of transcripts. While the genome is largely static, the transcriptome is highly dynamic and the cohort of expressed transcripts at a specific development stage or physiological condition will allow us to understand development and disease on a molecular level. However, two problems complicate the study of transcriptomes: 1) the presence of artefacts and biases and 2) pervasive transcription. In order to properly interpret transcriptional output, it is necessary to filter out the noise from the biological signal.

\subsection{Artefacts and biases in high-throughput sequencing}

Transcriptome profiling falls under the category of functional genomics, in which the aim is to identify functional elements in the genome by evidence of transcription. The "read out" from such experiments represents the expression of transcripts, which by itself is not entirely informative. The raw signal needs to be annotated or in order words, put into perspective, usually with respect to the genome. In the case of transcriptome profiling via high-throughput sequencing, the raw signal, i.e. sequenced reads, are first processed and mapped back to the genome. The processing step involves quality control steps that remove reads with missing features, ambiguously called bases, and reads that are composed of the primer or adaptor sequences used in the library preparation. Furthermore, different thresholds are usually implemented after read mapping to remove potentially spurious events; these thresholds can be based on the mapping confidence or mapping quality, and expression strength. (It is important to note that using various thresholds may remove potential signal from the data.) The mapped reads are then annotated to genomic features, such as protein-coding genes and repetitive elements, or to genomic properties, such as sequence conservation to other genomes and the nucleotide composition of a region. Finally, the annotated data is interpreted with respect to the experiment design; ``sanity checks", which are basic tests that can quickly assess the validity of the results, are commonly performed.

The identification of artefacts and biases is important prior to the analysis of data; while tools \citep{pmid19737799} are available for performing quality control steps, they are unable to deal with idiosyncratic biases from specific protocols. In chapter ~\ref{template_switching}, we demonstrated a particular type of bias that was introduced with the use of molecular barcodes. DNA barcodes are used to allow different RNA sources to be pooled at an early stage of sample preparation and are easily introduced by modifying the sequences of primers or linkers. This has two main benefits in that the starting material is increased and different samples can be sequenced on a single flowcell, increasing the cost-effectiveness of the experiment. In our study, we observed that the efficiency of primer annealing was dependent on the barcode sequence, thus creating a barcode bias. Furthermore, we noticed a particular type of artefact that was artificially produced during the template-switching step of the experimental protocol. Specifically, these template-switching artefact, which are a result of strand invasion, interrupted the process of reverse transcription when the cDNA has increased sequence complementary to the template-switching primer. These artefacts and biases created problems in the downstream analyses, as libraries prepared with the same or similar barcodes clustered together, i.e had similar expression profiles, regardless of the biological origin of the library. We could develop a bioinformatic solution to remove the artefacts, however, in the end the best solution was introducing a spacer sequencer into the template-switching primer. This ensured that the template-switching efficiency was equal in libraries prepared with different barcodes.

\subsection{Pervasive transcription and functional RNA products}

Another complication with interpreting transcriptional output is pervasive transcription, as it is difficult to distinguish between random and regulated transcriptional events. There have been two surprises in the genomics field since the completion of the draft human genome sequence \citep{lander2001initial,venter2001sequence}. The first was the number of genes in the human genome, which many genome biologists had over-estimated prior to the sequencing of the human genome. This came as a surprise, as it was thought that organismal complexity was a consequence of the number of genes; however, humans have roughly the same number of genes as \textit{Caenorhabditis elegans}. The next surprise was the observation that most of the human genome was transcribed \citep{pmid11988577, pmid17571346}, i.e. pervasive transcription. This phenomenon was first observed using tiling arrays \citep{pmid17510325} and later by various transcriptome profiling methods \citep{pmid22955620}. However, whether pervasive transcription consists mainly of background transcriptional noise or is of functional importance has been a matter of debate. The low fidelity of RNA polymerase II \citep{pmid17277804} and promiscuous binding of transcription factors \citep{pmid22868264} may give rise to ``leaky" transcription. Furthermore, the existence of various systems that suppress pervasive transcription \citep{pmid24267449} also raises the question of whether these products of pervasive transcription are functional.

Despite questions over their functionality, pervasive transcription is known to produce various classes of non-coding RNAs (ncRNAs) with characteristic features. These include multiple classes of small RNAs that are associated to promoters \citep{pmid19920851}, long non-coding RNAs (lncRNAs) \citep{pmid24290031}, and long intergenic non-coding RNAs (lincRNA) \citep{pmid23818866} (See table \ref{table:ncRNAs}). Evidence of context-specific transcription, e.g. transcription at a specific developmental stage, tissue type, or against a particular stimulus, has been observed in these classes of ncRNA. In chapter ~\ref{ddrna}, we profiled the small RNA population of cells that were induced for DNA damage, as a role for small RNAs in the DNA damage response (DDR) had been previously observed in \textit{Neurospora crassa} \citep{pmid19444217}. Our work and the work of an independent group, demonstrated that small RNAs could form in the vicinity of a double strand break \citep{francia2012site, pmid22445173}. These RNAs, which we termed as DDRNAs, had specific size and nucleotide profiles; we noted that 82.9\% of the DDRNAs started with an A or U (Supplementary figure 23d) and were enriched at lengths of 22-23 nt \citep{francia2012site}.

While the biogenesis of DDRNAs is unclear, knock-down of Dicer, Drosha \citep{francia2012site} and \textit{Ago2} \citep{pmid22445173} impaired the DNA repair efficiency. The size profile of DDRNAs match that of miRNAs and knock-down of the miRNA processing ribonucleases reduced the ratio of DDRNAs as well as the DDR \citep{francia2012site} suggesting that they may be processed by Dicer and Drosha. Many small RNAs are known to possess specific sequence biases at their 5' end and a rigid loop in the MID domain of human AGO2 allows specific contacts to a 5'-terminal U or A \citep{pmid23732335}, supporting the potential interaction of AGO2 with DDRNAs. As a signalling cascade is initiated from the site of DNA damage \citep{pmid19847258}, it was hypothesised that AGO2, which are known to bind to different classes of small RNAs, recruits the DNA damage repair complex \citep{pmid22445173}. However, as these nuclease enzymes require precursor RNAs to form the mature small RNAs, it is not clear if precursors exist or come into existence. A wild hypothesis, is that pervasive transcription functions as a surveillance system for DNA damage, since we observed DDRNAs even before the induced double strand break. Once there is DNA damage, these transcripts are processed into DDRNAs to signal the DDR. In addition, AGO2 may also play a role in the biogenesis of DDRNAs as only AGO2 is catalytically active amongst the Argonaute proteins in mammals, and can function as an endonuclease \citep{pmid23732335}.

\subsection{Expression of piRNAs in the absence of MECP2}

The guiding and recruitment of various complexes to specific genomic loci is becoming an emerging theme when discussing the potential role of various ncRNAs. Apart from the classical example of miRNAs being associated with the RNA-induced silencing complex, many lincRNAs have been shown to be bound to chromatin-modifying complexes and may function to guide these complexes to regions that need to undergo epigenetic regulation \citep{pmid21915889,pmid19571010}. Intriguingly, many enzymatic members of chromatin remodelling complexes do not contain DNA binding domains but possess RNA binding domains. For example, \textit{Xist} has been shown to recruit chromatin silencing proteins within the Polycomb complex to induce gene silencing via histone methylation \citep{pmid17869504}. Other ncRNAs including \textit{RepA}, \textit{Air}, \textit{Kcnq1ot1}, and \textit{Hotair} have also been shown to be associated with Polycomb and direct the complex to specific loci \citep{pmid21915889}. One class of small RNAs, known as piRNAs form RNA-protein complexes with piwi-type proteins, which are also related to the Argonaute family of proteins, and guides the complex to retrotransposons, which silences their activity. While piRNAs are known for their activity in germ cells, we showed that piRNAs are also expressed in the cerebellum. In chapter ~\ref{pirna}, we compared the expression of piRNAs in the cerebellum of normal and \textit{Mecp2} knock-out mice. Our hypothesis was that \textit{Mecp2} regulates the expression of retrotransposons \citep{pmid21085180} and in its absence will lead to the widespread expression of retrotransposons, which may lead to an increase in the number of piRNAs.

A recent study demonstrated that putative piRNAs map to lncRNAs, suggesting the possibility that some lncRNAs can act as piRNA precursors \citep{pmid24981367} and that piRNAs can regulate lncRNAs in the same manner that they regulate retrotransposons. Given that TEs make up a large portion of lncRNAs \citep{pmid23637635} it is not surprising that piRNAs have the ability to regulate lncRNAs. In another recent study, it was indeed demonstrated that piRNAs derived from TEs mediated the degradation of lncRNAs \citep{Watanabe05122014}. Given these observations, the mis-regulation of piRNAs may have further downstream effects. In chapter ~\ref{pirna}, we demonstrated that piRNAs were over-expressed in mice with a \textit{Mecp2} KO. Given that retrotransposons are capable of mobilising in neuronal progenitor cells from rodents \citep{pmid15959507} and humans \citep{pmid19657334} and are much more active in brain tissues \citep{pmid22037309}, the mis-regulation of piRNAs may affect normal neuronal activity. Our preliminary analyses suggests that it may be of interest to study the potential association of piRNAs to RTT.

\subsection{The repetitive human genome}

One of the major arguments against ascribing function to products of pervasive transcription is due to the fact that the human genome is largely occupied by repetitive DNA sequences, mainly made up of TEs. These elements have been historically considered as simply selfish products that have been able to propagate themselves very successfully \citep{doolittle1980selfish,orgel1980selfish}. Despite this, various functions have been ascribed to these elements; for example transcripts derived from TEs are able to regulate chromatin structure in centromeres and neocentromeres \citep{pmid19180186}. TEs may also serve a structural role as it was demonstrated that interspersed repeat sequences, such as LINE-1, are associated with euchromatin and loss of these repeat sequences caused aberrant chromatin distribution and condensation \citep{pmid24581492}. TEs may also act as CREs as a large number of transcriptional events has been shown to be initiated within TEs and served as alternative promoters \citep{pmid19377475} and TEs may act as TFBSs \citep{pmid18682548}. Studies have also demonstrated the association of TEs to lncRNAs \citep{pmid23181609, pmid23637635} and in fact it has been suggested that TEs contributed to the origin of many lncRNAs \citep{pmid25218058}. The presence of TEs within lncRNAs led to the Repeat Insertion Domains of LncRNAs (RIDLs) hypothesis \citep{pmid24850885}, which suggested that TEs act in a manner similar to structural domains in proteins.

In chapter ~\ref{repeat}, we profiled the expression of TEs in a large panel of samples (988 libraries), given the many potential functional roles of TEs. As previously reported, we observed that on average, expression from TEs are more tissue-specific and lowly expressed compared with protein-coding transcripts. The relatively low expression strength of pervasively transcribed products has been used to support the claim that these products are the consequence of degradation or background noise. However, it should be pointed out that unlike mRNAs, which require a longer half-life in order to exported into the cytoplasm for translation, ncRNA exert their function immediately in the nucleus. Many transcriptome profiling methods do not enrich for nuclear fraction, where many transcripts arising from TEs exist \citep{pmid24777452}. Furthermore, in the case that ncRNA are expressed in small quantities, like transcription factors, they are still able to trigger an amplified cascade of downstream events. However, despite their lower expression patterns, we were able to cluster biologically similar libraries together using the expression profiles from TEs. We also showed that transcriptional events initiating within TEs are overlap small RNAs, enhancers, and lincRNAs more often than chance. These lines of evidence support the notion that some TEs have become exapted in the human genome.
