\setlength{\parskip}{\baselineskip}%
\setlength{\parindent}{0pt}%

In the winter of 1868/9, Swiss physician and biologist, Johannes Friedrich Miescher isolated an unknown substance from the nuclei of cells\cite{dahm2008discovering}. This substance was unlike anything he had observed before; it was resistant to protease, lacked sulphur and contained large amounts of phosphorous. He recognised that he had isolated a novel substance to which he named nuclein. Today, we know of this substance as deoxyribonucleic acid (DNA), the molecule of heredity.

Although Miescher speculated that nuclein may have had a role in hereditary, he later rejected the idea. It wasn't until 1944, when Oswald Avery, Colin MacLeod, and Maclyn McCarty were able to characterise the transforming factor in Griffith's Experiment\cite{griffith1928significance} as DNA\cite{avery1944studies}. They performed a series of biochemical experiments that isolated DNA and then demonstrated that by using an enzyme that only degraded DNA (deoxyribonucleodepolymerase or DNase), the transforming power was lost. It was later confirmed in 1952 by Alfred Hershey and Martha Chase that DNA is indeed the genetic material by the use of bacteriophages\cite{hershey1952independent}.

Fast forward to the 21st century and not only have we sequenced the entire collection of DNA of our very own species \cite{venter2001sequence, lander2001initial}, we can sequence an entire human genome in a matter of days by using high-throughput sequencers. We have also just recently reached the \$1,000 genome era, whereby we can sequence the entire genome of an individual for around \$1,000 US dollars. In contrast, the Human Genome Project, which gave us the first glimpse of the human genome costed around 2.7 billion in fiscal year 1991 US dollars\cite{nhgri2010cost}. This dramatic drop in sequencing cost means that DNA sequencing is much more accessible to scientists and medical practitioners who wish to apply sequencing to their work.

In the journal Nature Reviews Genetics, there is a stream of articles on the applications of high-throughput sequencing\cite{applicationsofsequencing}. These include, but are not limited to, studying gene expression, DNA-protein interactions, non-protein coding RNAs, DNA methylation, structural variants, histone modifications, DNA mutations, single-cells, and populations of bacteria. This thesis consists of several applications of DNA sequencing towards understanding a range of specific biological questions.

\section{The central dogma}

DNA is a type of nucleic acid that consists of a base, a pentose, and a phosphate. The bases are aromatic ring (heterocyclic) compounds containing nitrogen and there are five main bases, which can be divided into purines and pyrimidines. There are two types of pentose sugars: ribose and 2-deoxyribose; compounds that consist of a base and a pentose are called nucleosides. In nucleosides, if phosphates are linked to the hydroxyl group of their sugar then they are called nucleotides or nucleic acids. This sugar-phosphate backbone forms the structural framework of nucleic acids. Nucleic acids with a ribose sugar are known as ribonucleic acid (RNA) and those with a 2-deoxyribose sugar are known as deoxyribonucleic acid (DNA).

The molecular basis of heredity is DNA.

The relationship between genes and proteins 

\subsection{Transcription}

The genome is a store of biological information but requires the coordinated activity of enzymes and proteins to bring about transcription. Chromatin structure and nucleosome positioning are altered in order for the transcriptional machinery to access parts of the genome for transcription. The pre-initiation complex that contains RNA polymerase II and general transcription factors forms near the core promoter region around the transcription start site and primes RNA polymerase II for transcription. Processing of precursor RNA include the addition of a cap and poly-A tail, splicing, and RNA editing. Removal of the cap is considered to the first step towards mRNA degradation. The capping procedure is thought to occur only in the nucleus, however a cytoplasmic form of a capping enzyme has been identified; the role of cytoplasmic re-capping.

\subsection{RNA polymerase}

\subsection{Reverse transcription}

\section{DNA sequencing}

\subsection{Sanger / Maxam-Gilbert sequencing}

\subsection{Sequencing by synthesis}

\section{Applications of sequencing}

The age of high throughput sequencing and the application of this technology to studying the transcriptome (also epigenome and interactome)

\subsection{Transcriptomics}

The transcriptome is the collection of RNA molecules derived from the genome at a particular time and location. The diversity of cells is determined by the expression patterns of genes and transcription factors (TFs).

\begin{itemize}

   \item Studies on the yeast transcriptome using DNA microarrays showed that although mRNAs are being degraded and re-synthesised all the time, the composition of the yeast transcriptome undergoes very little change if the environment remains constant
   \item However on switching from aerobic to anaerobic respiration, the levels of over 700 mRNAs increase by a factor of two or higher, and another 1,000 mRNAs decline to less than half their original amount
   \item Studying the transcriptome of acute lymphoblastic leukemia cells with respect to acute myeloid leukemia cells revealed differences that made it possible to distinguish the two types of leukemia
   \item Collection of full length cDNAs by the FANTOM consortium revealed many transcripts of unknown function (TUFs)
   \item The technologies that allowed full length cDNA sequencing
   \item Cap Analysis Gene Expression (CAGE) for genome-wide identification of transcription start sites
   \item Biotinylation of diol groups of RNA (High Efficiency Selection of Full-length cDNA by Improved Biotinylated Cap Trapper)
   \item PacBio sequencing has read lengths greater than 10kb and can be used for full length cDNA sequencing

\end{itemize}

\section{The repetitive genome}

Since the release of the draft human genome sequence\cite{venter2001sequence, lander2001initial}, it was established that only a small fraction of the genome is made up of protein-coding sequences and the majority of the genome was made up with repetitive elements. As the human genome contains the entire instruction set that is necessary for the development of a human, the decoding of the genome was thought to provide answers to many outstanding biological questions. However, there are several paradoxes that are still currently unresolved:

\begin{enumerate}
   \item K-value paradox: complexity does not correlate with the number of chromosomes
   \item C-value paradox: complexity does not correlate with genome size
   \item N-value paradox: complexity does not correlate with the number of protein coding genes
\end{enumerate}

If we measure organismal complexity in terms of mental cognition, we observe that complexity is not correlated to the number of chromosomes, the size of genomes, and the number of protein coding genes. Humans have 46 chromosomes and some species of butterflies have over hundreds of chromosomes, such as \textit{Polyommatus atlantica}. In terms of genome size, the species \textit{Polychaos dubium}, a freshwater amoeboid, has one of the largest genomes known with ~670 gb of DNA sequence; humans on the other hand have a genome size of ~3 gb. And lastly humans have roughly the same number of protein-coding genes as \textit{Caenorhabditis elegans}, ~20,000 versus ~21,000, respectively.


The \textit{C. elegans} genome is ~100 mb\cite{celegans1998sequencing}, which is ~30 times smaller than the human genome, despite having a similar number of genes. The difference in the genome size is mainly due to repetitive elements, which makes up roughly 50\% of the human genome.

The origin of the term "junk DNA" is usually attributed to Susumu Ohno, who used it to describe pseudogenes, which are gene copies that have no known biological function. In its modern day usage, "junk DNA" is used to describe DNA sequence that goes not play a functional role in an organism. Dr. Ohno estimated that there would be an upper limit to the number of functional loci in mammalian genomes based on mutational load and a fixed mutation rate. He predicted that mammalian genomes could not have more than 30,000 loci under selection as this would guarantee a progressive decline in fitness, leading to extinction.

In 1980, Leslie Orgel and Francis Crick, wrote that junk DNA has little specificity and conveys little or no selective advantage to the organism\cite{orgel1980selfish}.

W.Ford Doolittle and Carmen Sapienza wrote in 1980\cite{doolittle1980selfish}: "When a given DNA, or class of DNAs, of unproven phenotypic function can be shown to have evolved a strategy (such as transposition) which ensures its genomic survival, then no other explanation for its existence is necessary."

\begin{figure}[h]
  \centering
  \includegraphics[width=10cm, angle=270]{barplot_repeat_family.eps}
  \caption[Tally of repetitive element families in the hg19 genome]
   {The number of repetitive element families determined by RepeatMasker in the hg19 genome.}
\end{figure}

\begin{itemize}
   \item Messenger RNAs are transcripts of protein-coding genes and are translated into protein
   \item Classical non-coding RNA found in both prokaryotes and eukaryotes are ribosomal RNAs and transfer RNAs, which are both involved in protein synthesis
   \item Diverse class of non-coding RNAs, broadly broken down into long and short non-coding RNA
   \item Assembly of non-coding RNAs with proteins as ribonucleoprotein (RNP) structures
   \item Interaction of non-coding RNAs with chromatin
   \item Pervasive transcription describes the observation that a large percentage of DNA in mammalian genomes is transcribed.
   \item Tiling arrays and full length cDNA sequencing suggested that most of the genome is transcribed
   \item The ENCODE pilot project provided multiple lines of evidence that most of the mammalian genome is associated with at least one primary transcript, i.e., pervasive transcription of the genome
   \item Are the majority of detected low-level transcription due to technical artifacts and/or background biological noise?
   \item Protocols that capture only poly-A transcripts as opposed to poly-A minus and cytoplasmic versus nuclear enrichment libraries
   \item Sequencing depth and sampling of RNA molecules; absolute transcript quantification will help (such as using unique molecule identifiers and non-PCR based methods)
   \item Functional transcriptomics in the post-ENCODE era, specifically what is the criteria for functionality
   \item Repeat-associated binding sites (RABS) are over-represented in proximity of regulated genes and that the binding motifs within these repeats have undergone evolutionary selection
   \item Indeed, studies conducted both at the gene and genome levels have uncovered TE insertions that seem to have been co-opted - or exapted - by providing transcription factor binding sites (TFBSs) that serve as promoters and enhancers, leading to the hypothesis that TE exaptation is a major factor in the evolution of gene regulation.
   \item The transcription of repetitive elements, especially transposable elements, in specific tissues
   \item Repetitive elements are usually highly methylated, however differentiation methylation patterns of repetitive elements was observed in specific tissues
   \item Functions of some GWAS candidates in intergenic regions (such as http://www.nature.com/nature/journal/v507/n7492/full/nature13138.html)
\end{itemize}

\section{Stem cells}

\begin{itemize}
   \item Stem cells have two key characteristics that make them different from other cells
   \item They can reproduce themselves for long periods of time; this is known as self-renewal and
   \item They can differentiate or specialise into specific cell types under certain conditions
   \item Embryonic stem cells are derived from the inner cell mass of blastocysts. In normal development, the inner cell mass begins growing into all the different cell types of the fully developed body
   \item In our fully developed bodies, we have some self-renewing cells that reside in a specific tissue that are able to generate specific types of differentiated cells, i.e., they have limited differentiation ability. They were named adult stem cells because they reside in fully formed tissues.
   \item However, some adult stem cells have a transdifferentiation ability, which is that they can generate cells outside their own tissue type.
   \item They are also known as somatic stem cells or tissue stem cells.
   \item Haematopoietic stem cells (HSCs), which are somatic stem cells, reside in the medulla of the bone (bone marrow) and have the unique ability to give rise to all of the different mature blood cell types and tissues.
\end{itemize}
